src/main/resources/flattening/config/ssr_month/ssr.conf

```diff
@@ -0,0 +1,67 @@
+tables = [
```

Comment:
I think you can omit this file from the PR.

Reply:
I'm using it for the tests